test/functional/ui/screen_basic_spec.lua

```diff
@@ -690,4 +689,12 @@ describe('Screen', function()
       end)
     end)
   end)
+
+  it('nvim_ui_attach() handles very large width/height #2180', function()
+    screen:detach()
+    screen = Screen.new(999, 999)
```

Comment:
This takes ~1.5 seconds on a reasonably powerful machine (AppVeyor, and my local VM). If I bump it up to 9999x9999 luajit crashes with OOM. @jamessan should I mark this as "FRAGILE" so you can ignore it on debian CI?